espressif/vscode-esp-idf-extension
# Fix for issue #1096

## Description

Fixes error `Cannot read properties of undefined (reading 'curWorkspace')` when trying to run QEMU

Fixes #1096

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Steps to test this pull request

1. Click on "QEMU"
3. Observe error.

- Expected behaviour:

no error

- Expected output:

## How has this been tested?

tested on my local Windows

## Checklist
- [*] PR Self Reviewed
- [*] Applied Code formatting
- [*] Verified on Windows
